app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.java

```diff
@@ -417,6 +389,30 @@ private int howManyItemsToLoad() {
         return Math.max(MIN_ITEMS_INITIAL_LOAD, items);
     }
 
+    private String tryRetrieveVideoIdFromUrl(String url) {
+        try {
+            return youtubeStreamLinkHandler.getId(url);
+        } catch (Exception ex) {
+            if (DEBUG)
+                Log.d(TAG, "Failed to retrieve video ID from \"" + url + "\"", ex);
+            return null;
```

Comment:
Here we could probably show a Toast to the user, similarly to the one that's shown when a search result can't be parsed. It should contain the message "Sorry, some errors occoured" (the resource already exists in `strings.xml`) with a `REPORT` button.